docs/feature_combo.md

```diff
@@ -67,31 +86,100 @@ void process_combo_event(uint8_t combo_index, bool pressed) {
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
+This will send Ctrl+C if you hit Z and C, Ctrl+V if you hit X and V, and changel layer to `_LAYER` when you hold down S and D.  But you could change this to do stuff like play sounds or change settings.
 
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+# Advanced Configuration
+These configuration settings can be set in your keymaps `config.h`. 
 
-## Keycodes 
-
-You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+#### Keycodes 
+You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. The following keycodes are avalible for use in your `keymap.c`
 
 |Keycode   |Description                      |
 |----------|---------------------------------|
 |`CMB_ON`  |Turns on Combo feature           |
 |`CMB_OFF` |Turns off Combo feature          |
 |`CMB_TOG` |Toggles Combo feature on and off |
 
-## User callbacks
+#### Combo Term
+By default, the timeout for the Combos to be recognized is set to 50ms. This can be changed if accidental combo misfires are happening or if you're having difficulties pressing keys at the same time. For instance `#define COMBO_TERM 40` would set the time out period for combos to 40ms.
+
+#### Long Combos
+If you're using long combos or longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing. In this case, you can configure the size of the data structure used. Be aware, larger combo sizes will increase memory usage!
+
+| Keys | Define to be set     |
+|------|----------------------|
+| 8    | QMK Default          |
+| 16   | `#define EXTRA_LONG_COMBOS`  |
+| 32   | `#define EXTRA_EXTRA_LONG_COMBOS`|
+
+#### Modifier Combos
+If a combo resolves to a Modifier, the window for processing the combo can be extended independent from normal combos. By default this is disabled but can be enabled with `#define COMBO_MUST_HOLD_MODS`, and the time window can be configured with `#define COMBO_MOD_TERM 150` (default: 200). With `COMBO_MUST_HOLD_MODS` you cannot tap the combo any more which makes the combo less prone to misfires.
+
+#### Permissive Hold
+Similar to Tap-Hold configuration, Mod Combos can be setup to behave similarly. Add `#define COMBO_PERMISSIVE_HOLD` to your `config.h` to enable. This feature is extremely useful for fast typists as you don't have to wait for the Mod Combo's combo term to finish.
+
+If `COMBO_TERM` and `COMBO_MOD_TERM` are too close to each other this settings becomes less useful as this only affects if a key is pressed between `COMBO_TERM` and `COMBO_MOD_TERM`. All key presses under `COMBO_TERM` are always considered for overlapping combos.
```

Comment:
There's an extra 's' at the end of "this settings" and I can suggest replacing "affects if" by "takes effect if". To affect is generally followed by what it is being affected but here, what follows, is the condition in which it all *takes effect*.